git_machete/tests/functional/test_machete.py

```diff
@@ -79,7 +84,7 @@ def launch_command(*args: str) -> str:
         with io.StringIO() as out:
             with redirect_stdout(out):
                 cmd.launch(list(args))
-                cmd.flush_caches()
+                git_ctxt.flush_caches()
```

Comment:
Maybe just `git` instead of `git_ctxt`? same in cmd.py

This way it'll sound roughly as in command line: e.g. `git chechout <branch>` vs `git.checkout("<branch>")`